setup.sh

```diff
@@ -452,6 +452,91 @@ EOF
                 unset FILECONTENT_SSL_KEY_PEM
             fi
             ;;
+        FILECONTENT_LENSES_SSL_KEYSTORE)
+            if [[ -n $FILECONTENT_LENSES_SSL_KEYSTORE ]]; then
+                DECODE="cat"
+                if ! echo -n "$FILECONTENT_LENSES_SSL_KEYSTORE" | tr -d '\n' | grep -vsqE "$BASE64_REGEXP" ; then
+                    DECODE="base64 -d"
+                fi
+                $DECODE <<< "$FILECONTENT_LENSES_SSL_KEYSTORE" > /data/lenses.jks
+                chmod 400 /data/lenses.jks
+                cat <<EOF >>/data/lenses.conf
+lenses.ssl.keystore.location=/data/lenses.jks
+lenses.ssl.keystore.password="changeit"
+lenses.ssl.key.password="changeit"
+lenses.ssl.algorithm="SunX509"
+lenses.ssl.enabled.protocols="TLSv1.2"
```

Comment:
These two lines will override any setting that the user sets. We should skip them and uses Lenses' defaults.